resources/views/auth/register.blade.php

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
Es necesario mostrar al usuario cuando existen errores de validación.

Para esta iteración, basta con agregar el código que está en la documentación:
https://laravel.com/docs/9.x/validation#quick-displaying-the-validation-errors